README.md

```diff
@@ -142,7 +142,7 @@ Sample `config.json` file:
             "js": "{YOUR_APP_NAME}/{PATH_TO_YOUR_CONFIG_JS}"
         },
         "icon": "{YOUR_APP_NAME}/{PATH_TO_YOUR_ICON}",
-        "version": "{PLUGIN_VERSION}"/{PLUGIN_VERSION_NUMBER}
+        "version": "{PLUGIN_VERSION}"//{PLUGIN_VERSION_NUMBER}
```

Comment:
Do we need this sentence? I thought we don't need it because it's the same meaning of PLUGIN_VERSION.
> //{PLUGIN_VERSION_NUMBER}

Reply:
Maybe someone misunderstand about version and version number
ex: 
- version: "1.0.0" , "2.1.0" , "3.0.0" .... // string
- version number: 1, 2 ,3 / // integer
We need to add integer into this position, If there is no explanation i think someone can be added "1.0.0" string

Reply:
I see!
If so how about writing like this?

"version": {PLUGIN_VERSION} // plugin version number (integer) e.g. "version": 1

Reply:
I have updated